tests/test_modeling_gpt_neo.py

```diff
@@ -232,6 +230,86 @@ def create_and_check_gpt_neo_model_past(self, config, input_ids, input_mask, hea
         # test that outputs are equal for slice
         self.parent.assertTrue(torch.allclose(output_from_past_slice, output_from_no_past_slice, atol=1e-3))
 
+    def create_and_check_gpt_neo_model_attention_mask_past(
```

Comment:
Awesome new tests! Could we also add one/two fp16 tests? To make sure generation and forward pass works correctly :-)